src/averageAges.js

```diff
@@ -14,14 +14,40 @@
  *
  * @return {number}
  */
+
+function personAvgAge(male) {
+  const totalAges = male.reduce((sum, human) =>
+    sum + (human.died - human.born), 0);
+  const avgAge = totalAges / male.length;
+
+  return avgAge;
+}
+
+function filteredPeople(object, sex) {
+  const person = object.filter((human) => human.sex === sex);
+
+  function peopleWithDopInfo(dopInfo, callback) {
+    return person.filter((human) => (dopInfo)
+      ? callback(human)
+      : person);
+  }
+
+  return peopleWithDopInfo;
+}
```

Comment:
Why do you need to have a `person` array in closure? Your function called `filteredPeople` doesn't return filtered people at all. To make this work you can return an object with proper methods:
```js
function createPeopleFilter(people) {
  let persons = [...people];
  
  return {
    get list() {
      return persons;
    },
    applyFilter(filterFn) {
      persons = persons.filter(filterFn);
    },
  }
}


const men = createPeopleFilter(people)
  .applyFilter(({ sex }) => sex === 'm');

// apply any filters by calling `men.applyFilter()` method

console.log(men.list)
```